src/Components/Milestones/Carousel.jsx

```diff
@@ -1,69 +1,137 @@
-import React from 'react';
-import pic from './la.jpg';
-import pic1 from './chicago.jpg';
-import pic2 from './ny.jpg';
-import './carousel.css';
+import React, { useState, useEffect } from 'react';
+import { FaChevronLeft, FaChevronRight } from 'react-icons/fa';
+import { Grid, Slide, makeStyles } from '@material-ui/core';
+import PropTypes from 'prop-types';
+import CarouselSlide from './CarouselSlider';
+
+const useStyles = makeStyles(() => ({
+  arrows: {
+    height: '30px',
+    margin: '0',
+    cursor: 'pointer',
+    position: 'relative',
+    transform: 'translateY(25vh)',
+  },
+  slideshowDots: {
+    textAlign: 'center',
+    justifyContent: 'center',
+  },
+  slideshowDot: {
+    display: 'block',
+    height: '10px',
+    maxWidth: '10px',
+    margin: '26px 7px 0px',
+    borderRadius: '50%',
+    cursor: 'pointer',
+    backgroundColor: '#636363',
+  },
+  slideshowDotActive: {
+    display: 'block',
+    height: '10px',
+    maxWidth: '10px',
+    margin: '26px 7px 0px',
+    borderRadius: '50%',
+    cursor: 'pointer',
+    transform: 'scale(1.2)',
+    backgroundColor: 'white',
+    border: '1px solid #636363',
+  },
+}));
+
+const pic1 = require('../../Assets/chicago.jpg');
+const pic2 = require('../../Assets/ny.jpg');
+const pic3 = require('../../Assets/la.jpg');
+
+const SLIDE_INFO = [
+  { id: 1, title: 'chicago', image: pic1 },
+  { id: 2, title: 'la', image: pic2 },
+  { id: 3, title: 'ny', image: pic3 },
+];
+
+function Arrow(props) {
+  Arrow.propTypes = {
+    direction: PropTypes.string,
+    clickFunction: PropTypes.func,
+  };
+  Arrow.defaultProps = {
+    direction: 'left',
+    clickFunction: () => {
+      // do nothing
+    },
+  };
+  const { direction, clickFunction } = props;
+  const icon = direction === 'left' ? <FaChevronLeft /> : <FaChevronRight />;
+
+  return (
+    <div
+      onClick={clickFunction}
+      role="button"
+      tabIndex="0"
+      onKeyPress={clickFunction}
+    >
+      {icon}
+    </div>
+  );
+}
```

Comment:
Make a separate jsx file for this component

Reply:
✅